syntax = "proto3";
import "db_base.proto";
import "account_database.proto";

message account_database
{
  option(OptionTableName)  = "account";
  option(OptionPrimaryKey) = "account";
  string account = 1;
  string password = 2;
  account_simple_players simple_players = 3;
}
